Applications/CLI/Tests.cmake

```diff
@@ -126,7 +126,29 @@ if(NOT OGS_USE_MPI)
         )
         endforeach()
 
+    # Some Neumann BC tests
+    AddTest(
+        NAME NeumannBC_cube_top
+        PATH Elliptic/cube_1x1x1_NeumannBC
+        EXECUTABLE ogs
+        EXECUTABLE_ARGS cube_1e3_top_neumann.prj
+        WRAPPER time
+        TESTER vtkdiff
+        ABSTOL 1e-16 RELTOL 1e-16
+        DIFF_DATA cube_1e3_top_neumann_pcs_0_ts_1_t_1.000000.vtu pressure pressure
```

Comment:
After the fix AddTest PR cmake will complain about this change. So there will be no silent failure.